game-core/src/main/java/games/strategy/triplea/delegate/battle/BattleStepStrings.java

```diff
@@ -9,11 +9,15 @@
   String CASUALTIES_SUFFIX = " casualties";
   String LAND_PARATROOPS = "Land Paratroopers";
   String NAVAL_BOMBARDMENT = "Naval bombardment";
+  String NAVAL_BOMBARD = "naval bombardment";
```

Comment:
Hmm, okay. Though that seems to be overly context dependent. I wonder if it would be better to do these with built-in patterns and use them with string format. Alternatively maybe as method invocations even rather than contexts. The problem I see is that the usage is very specific, while this class is independent, it turns out to have a level of coupling to where it is used.

Reply:
All of these strings are only used for the Battle UI in the step list on the left hand side.  I think a probably better thing would be to use some sort of localization library and have these strings put in the language files.  I don't know if that is in any future plans.